src/seq/Seq.jl

```diff
@@ -188,12 +188,12 @@ function gap end
 
 gap(::Type{Char}) = '-'
 
-include("sequence.jl")
 include("nucleicacid.jl")
 include("aminoacid.jl")
 include("alphabet.jl")
 include("bitindex.jl")
-include("bioseq.jl")
+include("sequence.jl")
+include("bioseq/bioseq.jl")
 include("hash.jl")
 include("randseq.jl")
```

Comment:
hash.jl and randseq.jl are also tightly related to `BioSequence`, so it may be better to move them into the bioseq directory.